app/views/cycles/plugin_relations/petition.html.slim

```diff
@@ -26,105 +26,105 @@ section#petition-index
+              - if petition_past_pdf_versions.length > 0
+                div.medium-padded.subtitle.visible-lg-block
+                  a.past-versions href="javascript:void(0)" style="color: #{@cycle.color}"
+                    strong Veja versões anteriores
+
+                script#popover_template type="text/template"
+                  .petition-past-versions-popover.popover role="tooltip"
+                    .arrow
+                    h3.popover-title
+                    .popover-content
+
+                script#past-versions-popover type="text/template"
+                  ul.list-unstyled.petition-past-versions-list
+                    - petition_past_pdf_versions.each do |version|
+                      li.clearfix
+                        a.download href="#{version.pdf_url}" title="Clique para baixar essa versão"
+                          .version style="color: #{@cycle.color}"
+                            = "Versão #{version.version_name}"
+                          - if version.blockchain_timestamp
+                            = "Criada em #{version.blockchain_timestamp.strftime("%d/%m/%Y às %H:%M")}"
+
+        .row.phone-bottom-info.visible-xs-block
           - if @phase.in_progress?
-            .col-xs-6
+            .col-xs-6.progress
               - if @phase.remaining_days > 0
                 strong= "#{@phase.remaining_days} #{@phase.remaining_days > 1 ? "dias restantes" : "dia restante"} "
               - else
                 strong Último dia
               div
                 small para o encerramento
-          .col-xs-6
-            strong.total-signatures
+          .col-xs-6.text-right.signatures
+            .total-signatures
             div
               small assinaturas
 
-        .div.bottom-info-clear
-
-        .row#petition-signers-desktop
+        .row#petition-signers-desktop.signatures-list.visible-lg-block
         .row.view-all-signatures
-          a href="#{petition_signatures_path(@petition)}" style="color: #{@cycle.color}" Confira as listas de assinaturas autenticadas
+          .container-fluid
+            a href="#{petition_signatures_path(@petition)}" style="color: #{@cycle.color}" Confira as assinaturas autenticadas
 
-    .col-lg-6.petition-view.petition-body-container
+    .col-lg-6.col-md-12.petition-view.petition-body-container
       - if  @phase.plugin_relation.ready?
         div.embedded-link
           strong.pull-right style="color: #{@cycle.color}"
             i.icon-share
             |  Incorporar
-        p= @petition.presentation
+        p.presentation= @petition.presentation
         .row
           .col-xs-12
             = you_tube_embed @petition.video_id
         .row
           .col-xs-12
             h3.section-title.petition-body-title style="border-color: #{@cycle.color}" Projeto de lei
-        p.petition-body= markdown @petition.published_version.body
+        .petition-body= markdown @petition.published_version.body
```

Comment:
yeah hahah and changing that was also the way to solve that crazy nesting error that I talked about on slack